### ISSUE TITLE
When I open a message with a photo, it does not load the photo correctly in full screen if I do it from a web browser

### ISSUE THREAD

**Issue by Mgil10** [flyerhq/flutter_chat_ui#155]
# General

## What bug do you experience? 🐞
When opening a message with a photo, it does not load the photo correctly in full screen, it lets interact with the close button and scroll the photo down to close it, if I do it from a web browser, this works correctly in android.

## How can it be reproduced? 🤔

Step 1. Run the sample code in a web browser from https://docs.flyer.chat/flutter/chat-ui/advanced-usage
Step 2. Send an image.
Step 3. Click on the image.

## What behavior is expected? 💡
I was expecting the same thing to happen as on Android, that the photo opens in full screen correctly
<br />

# Extras

## Screenshots or videos 📸
If applicable, add screenshots or videos to help explain your problem.

https://streamable.com/qyoaq5

## Code snippets 📝
If applicable, add code samples to help explain your problem.

```dart
class StartChat extends StatefulWidget {
  const StartChat({Key? key}) : super(key: key);

  @override
  _StartChatState createState() => _StartChatState();
}

class _StartChatState extends State<StartChat> {

  final List<types.Message> _messages = [];

  final _user = const types.User(id: '06c33e8b-e835-4736-80f4-63f44b66666c');

  void _addMessage(types.Message message) {
    setState(() {
      _messages.insert(0, message);
    });
  }

  void _handleSendPressed(types.PartialText message) {
    final textMessage = types.TextMessage(
      author: _user,
      createdAt: DateTime.now().millisecondsSinceEpoch,
      id: randomString(),
      text: message.text,
    );

    _addMessage(textMessage);
  }

  void _handlePreviewDataFetched(
    types.TextMessage message,
    types.PreviewData previewData,
  ) {
    final index = _messages.indexWhere((element) => element.id == message.id);
    final updatedMessage = _messages[index].copyWith(previewData: previewData);

    WidgetsBinding.instance?.addPostFrameCallback((_) {
      setState(() {
        _messages[index] = updatedMessage;
      });
    });
  }

  void _handleImageSelection() async {
    final result = await ImagePicker().pickImage(
      imageQuality: 70,
      maxWidth: 1440,
      source: ImageSource.gallery,
    );

    if (result != null) {
      final bytes = await result.readAsBytes();
      final image = await decodeImageFromList(bytes);

      final message = types.ImageMessage(
        author: _user,
        createdAt: DateTime.now().millisecondsSinceEpoch,
        height: image.height.toDouble(),
        id: randomString(),
        name: result.name,
        size: bytes.length,
        uri: result.path,
        width: image.width.toDouble(),
      );

      _addMessage(message);
    }
  }

  
  @override
  Widget build(BuildContext context) {
    return SafeArea(
      bottom: false,
      child: Chat(
          theme: const DefaultChatTheme(),
          messages: _messages,
          onSendPressed: _handleSendPressed,
          user: _user,
          onPreviewDataFetched: _handlePreviewDataFetched,
          onAttachmentPressed: _handleImageSelection,
        ),
    );
  }
}

String randomString() {
  final random = Random.secure();
  final values = List<int>.generate(16, (i) => random.nextInt(255));
  return base64UrlEncode(values);
}
```

<br />

# Environment info

Please specify the flutter, flutter-chat-ui versions.

flutter: 2.5.3
flutter-chat-ui: ^1.5.1

`flutter doctor -v` output 👇

```sh 
[√] Flutter (Channel stable, 2.5.3, on Microsoft Windows [VersiÃ³n 10.0.19043.1320], locale es-ES)
    • Flutter version 2.5.3 at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (4 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at C:\Users\--\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.7.7)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.7.30621.155
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.62.1)
    • VS Code at C:\Users\--\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.28.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [VersiÃ³n 10.0.19043.1320]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 95.0.4638.69
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 95.0.1020.44

• No issues found!
```

<br />

# Platform

Device (e.g. Android emulator, iOS simulator, Samsung Galaxy S21): Chrome or Edge

OS version (e.g. iOS 14.5, Android 11): ?

<br />

<!-- Thank you for making us better 🤝--!>

**Comment by demchenkoalex** [comment]
Appreciate the perfectly created issue @Mgil10 :)

We are using another open source library for the image view, so I am not sure if we will be able to fix this on our side, but maybe we did something wrong, we can double check.

**Comment by demchenkoalex** [comment]
Fixed in [v1.5.2](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.5.2).
